src/mocks/yokai-watch-1/bugs.js

```diff
@@ -0,0 +1,123 @@
+		description: '',
+		sell: '1.56',
+		rareSell: '3.12'
+	},
+	{
+		name: 'Firefly (YW1)',
+		location: 'Flies around rivers and lakes. Can be found at night.',
+		description: '',
+		sell: '7.80',
+		rareSell: '15.60'
+	},
+	{
+		name: 'Flower Scarab (YW1)',
+		location: 'Found sometimes wherever there are bushes.',
+		description: '',
+		sell: '0.96',
+		rareSell: '1.92'
+	},
+	{
+		name: 'Grasshopper (YW1)',
+		location: 'Found wherever there are bushes.',
+		description: '',
+		sell: '0.75',
+		rareSell: '1.50'
+	},
+	{
+		name: 'Green Cicada (YW1)',
+		location: 'In the trees around business districts and residential areas. It's one of the bugs that appears the mostWhether it's day or night.',
+		description: '',
+		sell: '0.54',
+		rareSell: '1.08'
+	},
+	{
+		name: 'Ladybug (YW1)',
+		location: 'Found sometimes in any kind of bush.',
+		description: '',
+		sell: '0.60',
+		rareSell: '1.20'
+	},
+	{
+		name: 'Locust (YW1)',
+		location: 'Found wherever there are bushes.',
+		description: '',
+		sell: '2.22',
+		rareSell: '4.44'
+	},
+	{
+		name: 'Long-Horn (YW1)',
+		location: 'Found in various trees.',
+		description: '',
+		sell: '1.68',
+		rareSell: '3.36'
+	},
+	{
+		name: 'Praying Mantis (YW1)',
+		location: 'Found sometimes wherever there are bushes.',
+		description: '',
+		sell: '1.83',
+		rareSell: '3.66'
+	},
+	{
+		name: 'Rhino Beetle (YW1)',
+		location: 'Found rarely in the trees of Mt. Wildwood and trees elsewhere. Easier to find at night.',
+		description: '',
+		sell: '15.00',
+		rareSell: '30.00'
+	},
+	{
+		name: 'Sawtooth Stag (YW1)',
+		location: 'Found rarely in the trees of Mt. Wildwood and trees elsewhere. Easier to find at night.',
+		description: '',
+		sell: '5.40',
+		rareSell: '10.80'
+	},
+	{
+		name: 'Swallowtail (YW1)',
+		location: 'Found sometimes in the trees around town. Easier to find during the day.',
+		description: '',
+		sell: '1.41',
+		rareSell: '2.82'
+	}
+]
+	{
+		name:'Ladybug (YW1)', location:'Found sometimes in any kind of bush.', description:'', sell:'0.60', rareSell:'1.20'
+	},
+	{
+		name:'Locust (YW1)', location:'Found wherever there are bushes.', description:'', sell:'2.22', rareSell:'4.44'
+	},
+	{
+		name:'Long-Horn (YW1)', location:'Found in various trees.', description:'', sell:'1.68', rareSell:'3.36'
+	},
+	{
+		name:'Praying Mantis (YW1)', location:'Found sometimes wherever there are bushes.', description:'', sell:'1.83', rareSell:'3.66'
+	},
+	{
+		name:'Rhino Beetle (YW1)', location:'Found rarely in the trees of Mt. Wildwood and trees elsewhere. Easier to find at night.', description:'', sell:'15.00', rareSell:'30.00'
+	},
+	{
+		name:'Sawtooth Stag (YW1)', location:'Found rarely in the trees of Mt. Wildwood and trees elsewhere. Easier to find at night.', description:'', sell:'5.40', rareSell:'10.80'
+	},
+	{
+		name:'Swallowtail (YW1)', location:'Found sometimes in the trees around town. Easier to find during the day.', description:'', sell:'1.41', rareSell:'2.82'
+	}
+]
```

Comment:
can you add this at the end:

```js
export default bugs;
```

The same thing for the rest of the files, thx